boards/reel/doc.txt

```diff
@@ -0,0 +1,14 @@
+/**
+@defgroup    boards_reel reel
```

Comment:
The board group name should be more explicit, e.g. something like `Phytec reel board` or something. Otherwise, this will just add an entry called 'reel' in the doxygen documentation, this will look weird.

Regarding the build system, I know 'reel' is fast and simple to type when calling make, but it's also a too generic word. What about using `phytec-reel` ? Like this it will be listed close to other phytec boards (pba-xxx-zz-kw2x and phynode-kw41z).
This second comment is not blocking, and after all, you are the author, so you decide ;)